backend/database/schema.sql

```diff
@@ -3,73 +3,73 @@ CREATE TABLE user (
   lastname VARCHAR(100) NOT NULL,
   firstname VARCHAR(100) NOT NULL,
   email VARCHAR(255) NOT NULL,
+  /* frontend sends "Americas", "France", "Lebanon", "Philippines" or "West Africa" */
```

Comment:
Is that really mandatory to keep all these comments in the middle of your queries?

Reply:
At least until the options are actually available in front, it felt like they were easier to access like that? I can change that, it's just convenient.